app/js/index.js

```diff
@@ -26,6 +26,12 @@ $(document).ready(function() {
 
 
 function init() {
+  // don't initialize until user has uploaded a .csv file
+  if(document.getElementById('input_csv').files[0] == null){
+    console.log("Please upload a file");
```

Comment:
oh yeah my bad, that was really just for debugging, there should probably be some HTML to tell the user they need to upload the file.